lib/Core/Site/QueryType/Location/Fetch.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Netgen\EzPlatformSiteApi\Core\Site\QueryType\Location;
+
+use Netgen\EzPlatformSiteApi\Core\Site\QueryType\Location;
+use Symfony\Component\OptionsResolver\OptionsResolver;
```

Comment:
Not needed anymore.

Reply:
Removed in https://github.com/netgen/ezplatform-site-api/pull/70/commits/fcb80b0326fedcfeb5f6a50e610b4fae02ba4385